### PR TITLE
⚙️ DuckDuckGo Presence Updates

### PR DESCRIPTION
**Pull request by Cairo2k18**
Using TypeScript and PreMiD Settings.

Examples (proof it's actually working - note: added stock search information)
image
image
image
image
image
image
image

### CODE REVIEW

File: DuckDuckGo/dist/metadata.json [hunk 1]
```diff
@@ -8,10 +8,96 @@
...
   "tags": ["internet", "browing", "search"],
-  "category": "other"
+  "category": "other",
+  "settings": [
+    {
+        "id": "homepageMessage",
+        "title": "Homepage Message",
+        "icon": "fas fa-home",
+        "value": "Chilling on the homepage",
+        "placeholder": "Chilling on the homepage"
```
Comment (Bas950) [inline DuckDuckGo/dist/metadata.json:23]: Make it more generel as default like, Viewing the homepage, or Browsing the homepage

File: DuckDuckGo/dist/metadata.json [hunk 2]
```diff
@@ -8,10 +8,96 @@
...
+        "icon": "fas fa-cogs",
+        "value": "Changing my settings!",
+        "placeholder": "Changing my settings!"
+    },
+    {
+        "id": "standardSearch",
+        "title": "Standard Search Format",
+        "icon": "fas fa-search",
+        "value": "Searching the interwebz for %search%",
+        "placeholder": "Searching the interwebz for %search%"
```
Comment (Bas950) [inline DuckDuckGo/dist/metadata.json:37]: Same here, searching for: (not interwebz)

File: DuckDuckGo/dist/metadata.json [hunk 3]
```diff
@@ -8,10 +8,96 @@
...
+        "icon": "fas fa-shopping-bag",
+        "value": "Browsing the shops for %search%",
+        "placeholder": "Browsing the shops for %search%"
+    },
+    {
+        "id": "recipeSearch",
+        "title": "Recipe Search Format",
+        "icon": "fas fa-utensils",
+        "value": "Finding some yummy recipies for %search%",
+        "placeholder": "Finding some yummy recipies for %search%"
```
Comment (Bas950) [inline DuckDuckGo/dist/metadata.json:79]: remove yummy, before they turn out to be bad lmao xD

Comment (Cairo2k18) [reply DuckDuckGo/dist/metadata.json:79]: oh god I forgot about that one lmfao

**Pull request merged**